phpunit.xml.dist

```diff
@@ -38,8 +38,8 @@
             <file>src/Psalm/Internal/Provider/ParserCacheProvider.php</file>
         </exclude>
         <report>
-            <clover outputFile="build/logs/clover.xml"/>
-            <html outputDirectory="build/logs/phpunit-html/"/>
+            <!-- <clover outputFile="build/logs/clover.xml"/> -->
+            <!-- <html outputDirectory="build/logs/phpunit-html/"/> -->
```

Comment:
Newer paratest reads phpunit.xml and tries to merge coverage reports, even when none could be generated.